### DISCUSSION TITLE
ender 3 s1 support

### DISCUSSION THREAD

**Discussion by lenir67** [supermerill/SuperSlicer#2312] [Q&A]
will there be support for the s1?  I'm new I'm sure there's a way to make it work but yea i don't know what i am doing lol

**Comment by supermerill** [comment]
Have to write a tutorial to create & modify vendor profiles.

**Reply by lenir67** [reply]
that would be awesome

**Reply by thedroidkid** [reply]
Yeah I'm really wishing there was a stock profile for the s1 I see it in Prusa slicer but I can't seem to export it

**Comment by neophyl** [comment]
If the profile is in Prusa Slicer, then you can select it (as well as a print and filament profile).  You can then either use export Config or simply save a project.

If saving a config then you can use the saved config and import it into Super Slicer.

If opening a project file then you would have to check the settings for the printer have been correctly interpreted and then once you are happy they are you can then just save the profile as you would any other custom profile.  Might not be part of the vendor space anymore but all the settings would be there.
